modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/AlterTableAddColumnCommand.java

```diff
@@ -53,15 +53,17 @@ public static AlterTableAddColumnCommandBuilder builder() {
      *
      * @param tableName Name of the table to add new columns to. Should not be null or blank.
      * @param schemaName Name of the schema the table of interest belongs to. Should not be null or blank.
+     * @param ifTableExists Flag indicating whether the {@code IF EXISTS} was specified.
      * @param columns List of the columns to add to the table. There should be at least one column.
      * @throws CatalogValidationException if any of restrictions above is violated.
      */
     private AlterTableAddColumnCommand(
             String tableName,
             String schemaName,
+            boolean ifTableExists,
             List<ColumnParams> columns
     ) throws CatalogValidationException {
-        super(schemaName, tableName);
+        super(schemaName, tableName, ifTableExists);
```

Comment:
it`s strange to see such a flag here.

Reply:
I don't fully understand you suggestion.

It's valid statement
```
alter table if exists NON_EXISTING_TABLE add SOMECOLUMN int
```
should not produce updates and errors (in case NON_EXISTING_TABLE does not exist)

Reply:
i didn`t see if exists in standard for such a case : 11.10 <alter table statement>
i miss smth ?

Reply:
I think this is out of scope of this PR. At least PG accepts such syntax.

Btw, do you see `if exists` for `DROP TABLE` for example or `if not exists` for `CREATE TABLE` in the specification?